test.js

```diff
@@ -10,43 +10,52 @@ test('default', async t => {
 });
 
 test('full path', async t => {
-	t.is((await ghGot('https://api.github.com/users/sindresorhus')).body.login, 'sindresorhus');
+	t.is((await ghGot('https://api.github.com/users/sindresorhus', {prefixUrl: ''})).body.login, 'sindresorhus');
 });
 
 test('accepts options', async t => {
 	t.is((await ghGot('users/sindresorhus', {})).body.login, 'sindresorhus');
 });
 
-test('accepts options.endpoint without trailing slash', async t => {
-	t.is((await ghGot('users/sindresorhus', {endpoint: 'https://api.github.com'})).body.login, 'sindresorhus');
+test('accepts options.prefixUrl without trailing slash', async t => {
+	t.is((await ghGot('users/sindresorhus', {prefixUrl: 'https://api.github.com'})).body.login, 'sindresorhus');
 });
 
-test('dedupes slashes', async t => {
-	t.is((await ghGot('/users/sindresorhus', {endpoint: 'https://api.github.com/'})).body.login, 'sindresorhus');
+// This should be fixed in Got
+test.failing('dedupes slashes', async t => {
+	t.is((await ghGot('/users/sindresorhus', {prefixUrl: 'https://api.github.com/'})).body.login, 'sindresorhus');
 });
 
 test.serial('global token option', async t => {
 	process.env.GITHUB_TOKEN = 'fail';
-	await t.throwsAsync(ghGot.recreate()('users/sindresorhus'), 'Bad credentials (401)');
+	await t.throwsAsync(ghGot.recreate()('users/sindresorhus'), {
```

Comment:
I think it's more readable to put both on a new line.
```suggestion
	await t.throwsAsync(
		ghGot.recreate()('users/sindresorhus'),
		{
```